hack/benchmark/internal/operation/insert.go

```diff
@@ -0,0 +1,127 @@
+package operation
+
+import (
+	"context"
+	"io"
+	"strconv"
+	"sync"
+	"sync/atomic"
+	"testing"
+
+	"github.com/vdaas/vald/apis/grpc/v1/payload"
+	"github.com/vdaas/vald/hack/benchmark/internal/assets"
+	"github.com/vdaas/vald/internal/net/grpc/codes"
+	"github.com/vdaas/vald/internal/net/grpc/status"
+)
+
+func (o *operation) Insert(b *testing.B, ctx context.Context, ds assets.Dataset) (insertedNum int) {
+	b.ResetTimer()
+	b.Run("Insert", func(b *testing.B) {
+		req := &payload.Insert_Request{
+			Vector: &payload.Object_Vector{},
+			Config: &payload.Insert_Config{
+				SkipStrictExistCheck: false,
+			},
+		}
+		b.ResetTimer()
+
+		for i := 0; i < b.N; i++ {
+			v, err := ds.Train(i % ds.TrainSize())
+			if err != nil {
+				b.Error(err)
+				continue
```

Comment:
I think we should fail the test if the dataset return error to avoid incorrect benchmark result
```suggestion
				b.Fatal(err)
```